main.py

```
@@ -1107,6 +1509,285 @@ async def _sync_process(update: Update, context: ContextTypes.DEFAULT_TYPE, pack
+        await update.message.reply_text(
+            text, parse_mode="HTML", reply_markup=home_keyboard()
+        )
+        return
+
+    text = (
+        f"🗂 <b>DRAFT VAULT</b>\n{DIV}\n\n"
+        f"You have <b>{len(drafts)}</b> pending draft(s).\n\n"
+        "<i>Use the buttons below to approve, retry, or reject each draft.</i>"
+    )
+    keyboard_rows = []
+    for draft_id, file_id, status, created_at, expires_at in drafts:
```

Ruff (0.15.7)

[warning] Loop control variable `file_id` not used within loop body

Rename unused `file_id` to `_file_id`

(B007)